packages/core/core/src/utils/fetch.ts

```diff
@@ -14,11 +14,13 @@ export const createStrapiFetch = (strapi: Core.Strapi): Modules.Fetch.Fetch => {
     return fetch(url, fetchOptions);
   }
 
-  const globalProxy =
-    strapi.config.get<ConstructorParameters<typeof ProxyAgent>[0]>('server.globalProxy');
+  const proxy =
+    strapi.config.get<ConstructorParameters<typeof ProxyAgent>[0]>('server.proxy.fetch') ||
```

Comment:
I was wondering if it will be clearer to add a type alias for this `ConstructorParameters<typeof ProxyAgent>[0]`
will leave it for you to decided otherwise looks good